config/runtime/kbuild.jinja2

```diff
@@ -101,6 +101,16 @@ class Config:
                 print(f"[_merge_frags] Failed to merge fragment")
             print(f"[_merge_frags] Merging fragment done")
             os.remove(f"/tmp/{i}.config")
+    
+    def olddefconfig(self, srcdir):
+        cwd = os.getcwd()
+        os.chdir(srcdir)
+        print(f"[_olddefconfig] Running olddefconfig")
+        cmd = f"make ARCH=x86_64 olddefconfig"
+        ret = subprocess.run(cmd, shell=True).returncode
+        if ret != 0:
+            print(f"[_olddefconfig] Failed to run olddefconfig")
+        print(f"[_olddefconfig] Running olddefconfig done")
```

Comment:
That's a lot of debug noise though, maybe it's useful right now but it should really be cleaned up at some point.  The job log should contain all the necessary information.

Reply:
Yes, it is just debug at moment to get build steps right